src/mod/irc.mod/irc.c

```diff
@@ -840,6 +852,16 @@ static void check_tcl_kick(char *nick, char *uhost, struct userrec *u,
                  MATCH_MASK | BIND_USE_ATTR | BIND_STACKABLE);
 }
 
+static void check_tcl_invite(char *nick, char *from, char *invitee, char *chan)
+{
+  Tcl_SetVar(interp, "_invite1", nick, 0);
+  Tcl_SetVar(interp, "_invite2", from, 0);
+  Tcl_SetVar(interp, "_invite3", invitee, 0);
+  Tcl_SetVar(interp, "_invite4", chan, 0);
+  check_tcl_bind(H_invt, from, 0, " $_invite1 $_invite2 $_invite3 $_invite4",
```

Comment:
adjust this to match against `"<channel> <invitee>"`

Reply:
Done

Reply:
Done